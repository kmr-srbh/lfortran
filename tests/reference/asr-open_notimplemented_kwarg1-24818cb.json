{
    "basename": "asr-open_notimplemented_kwarg1-24818cb",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/warnings/open_notimplemented_kwarg1.f90",
    "infile_hash": "7571e57fc2000c98f3c9509df26eb64375f1393ab142090b7a86e7a2",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-open_notimplemented_kwarg1-24818cb.stdout",
    "stdout_hash": "f22d1e21bd03cb3281ccd31a4f7496997f4f7f0ecb55efbc44007f78",
    "stderr": "asr-open_notimplemented_kwarg1-24818cb.stderr",
    "stderr_hash": "f49630a5e5c44814fc6e999dac6118f4eaf6647ba2c9493155fc7b49",
    "returncode": 0
}