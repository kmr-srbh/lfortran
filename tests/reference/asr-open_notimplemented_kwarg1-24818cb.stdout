(TranslationUnit
    (SymbolTable
        1
        {
            open_notimplemented_kwargs:
                (Program
                    (SymbolTable
                        2
                        {
                            
                        })
                    open_notimplemented_kwargs
                    []
                    [(FileOpen
                        0
                        (IntegerConstant 10 (Integer 4) Decimal)
                        (StringConstant
                            "file_01_data.txt"
                            (Character 1 16 () PointerString)
                        )
                        (StringConstant
                            "old"
                            (Character 1 3 () PointerString)
                        )
                        ()
                    )]
                )
        })
    []
)
